### DISCUSSION TITLE
Find a way to count users

### DISCUSSION THREAD

**Discussion by leonlatsch** [leonlatsch/Photok#289] [Ideas]
Please feel free to submit your ideas on how to count users of Photok. As of F-Droid restrictions we cannot use Google Analytics or other proprietary services.

**Comment by WolfganP** [comment]
Besides what I suggested in https://github.com/leonlatsch/Photok/issues/277#issuecomment-1988546909 this site lists privacy oriented analytics platforms https://usermaven.com/blog/privacy-friendly-analytics#which-analytics-platforms-are-privacy-friendly , not many with free plans, unless you're considering self hosting some open source solution.

**Comment by leonlatsch** [comment]
Just found out about https://posthog.com/

Will need to see if its accepted by f-droid

**Comment by leonlatsch** [comment]
Actually https://plausible.io/ seems to be selfhosted aswell

**Comment by WolfganP** [comment]
> Actually https://plausible.io/ seems to be selfhosted aswell

I remember having read about [plausible selfhosting](https://github.com/plausible/community-edition/) experience in a blog, and traced it to [here](https://esc.sh/blog/plausible-analytics-selfhosting/) and  [here](https://www.reddit.com/r/selfhosted/comments/o8r293/i_tried_plausible_self_hosted_analytics_and_its/) . Maybe worth reading all experiences!

**Comment by leonlatsch** [comment]
After digging a bit, the best options seem to be:
- Matomo
- Umami (OpenSource)
- PostHog (OpenSource)

All of them are self hosted.

Although, none of them have a SDK, so this would need to be integrated manually via http.

**Comment by Fau57** [comment]
I feel very confused that no one has said or even asked about integration on competition with pixiv?

**Reply by leonlatsch** [reply]
What is pixiv and what do you mean by integration with pixiv?

**Comment by leonlatsch** [comment]
I have made a decision for now to go with TelemetryDeck.com .

They have a free plan, open source SDKs and do real anonymous telemetry to the point that its theoretically not even necessary to mention it anywhere legally.

To see how this is implemented and especially how this behaves on F-Droid. See the pr #659
